modules/core/src/v2/wallet.ts

```diff
@@ -1055,7 +1055,7 @@ export class Wallet {
       params = params || {};
       common.validateParams(params, ['address'], ['walletPassphrase', 'xprv', 'otp'], callback);
 
```

Comment:
nit: Add comment explaining this code path
```suggestion
// The sweep API endpoint is only available to utxo-based coins
```